src/utils/cookie.utils.ts

```diff
@@ -2,6 +2,8 @@ import { CookieOptions } from 'express'
 
 import environment from '../environment'
 
+export const refreshToken: string = 'refreshToken'
```

Comment:
```suggestion
export const refreshToken = 'refreshToken'
```

You don't need the type declaration on explicit assignments because it's smart enough to realize you're assigning it to a string.

Reply:
It's not necessarily wrong to type everything. Just not my style. If people prefer everything to be absolutely 100% have a type (ew), I believe there's a setting we can turn on to do that.

Reply:
i have no preference either way, just what i was taught haha. though having consistency either way would be good.